### PR TITLE
Automation tests for Hop 

### PR DESCRIPTION
**Pull request by lydia-karungi**
# :books: Linear Ticket :books:

### CODE REVIEW

File: .mocharc.yml [hunk 1]
```diff
@@ -5,11 +5,11 @@ full-trace: true
 color: true
 inline-diffs: true
 async-only: true
-parallel: true
+parallel: false
```
Comment (beemi) [inline .mocharc.yml:8]: don't commit this file

File: .gitignore [hunk 2]
```diff
@@ -7,6 +7,8 @@ build
 .env.local
 .env.*.local
 
+test/.env
```
Comment (beemi) [inline .gitignore:10]: ?

File: test/e2e/hop_tests/hop_swaps.spec.js [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+
+const testUtil = new TestUtil()
+const overviewPage = new OverviewPage()
+const homePage = new HomePage()
+const passwordPage = new PasswordPage()
+const swapPage = new SwapPage()
+
+let browser, page
+
+describe.only('SWAP feature["MAINNET"]', async () => {
```
Comment (beemi) [inline test/e2e/hop_tests/hop_swaps.spec.js:18]: .only

**Review by beemi — CHANGES_REQUESTED** [review]

File: .mocharc.yml (hunk 1 above)
Comment (lydia-karungi) [reply .mocharc.yml]: @beemi I have resolved this

File: .gitignore (hunk 2 above)
Comment (lydia-karungi) [reply .gitignore:10]: @beemi Not sure what you mean here

File: test/e2e/hop_tests/hop_swaps.spec.js (hunk 3 above)
Comment (lydia-karungi) [reply test/e2e/hop_tests/hop_swaps.spec.js]: @beemi this has been resolved

**Pull request merged**